index.html

```diff
@@ -847,90 +847,77 @@ <h2>Examples</h2>
-          "name": "mySensor",
-          "url": "http://myservice.org/mySensor/description"
-        }';
-        WoT.fetch("http://myservice.org/mySensor/description")
-        .then(td => {
-          let e_thing = WoT.produce(td);
-          // properties, actions and events have been added based on the TD
-          // now add the requests handlers
-          try {
-            e_thing.setPropertyReadHandler(propertyName => {
-              console.log("Handling read request for " + propertyName);
-              return new Promise((resolve, reject) => {
-              if (this.hasOwnProperty(propertyName))
-                resolve(this[propertyName]);
-              else reject('Property not found');
-            });
-            e_thing.start();
-          } catch(error) {
-            console.log("Error");
-          }
+      <pre class="example highlight" title="Create a new exposed Thing from a Thing Description">
+        let thingDescription = '{ "@context": [ "https://w3c.github.io/wot/w3c-wot-td-context.jsonld", "https://w3c.github.io/wot/w3c-wot-common-context.jsonld" ], "@type": [ "Thing", "Sensor" ], "name": "mySensor", "geo:location": "testspace", "interaction": [ { "@type": [ "Property", "Temperature" ], "name": "prop1", "schema": { "type": "number" }, "saref:TemperatureUnit": "degree_Celsius" } ] }';
+        try {
+          // note that produce() fails if thingDescription contains error
+          let thing = WoT.produce(thingDescription);
+          // Interactions were added from TD
+          // add server functionality (a Property handler specifically for prop1)
+          thing.setPropertyReadHandler( () => {
+            console.log("Handling read request for prop1");
+            return new Promise((resolve, reject) => {
+              resolve(Math.random(100));
+            })
+          }, "prop1");
+          thing.start();
+        } catch(err) {
+          console.log("Script error: " + err);
         }
-        });
       </pre>
 
-      <pre class="example highlight" title="Create a new exposed Thing from a Thing Description">
-        let thingDescription = '{
-          "name": "mySensor",
-          "description": {
-            "@context": [
-              "http://w3c.github.io/wot/w3c-wot-td-context.jsonld",
-              "http://w3c.github.io/wot/w3c-wot-common-context.jsonld",
-            ],
-            "@type": [ "Thing" ],
-            "interactions": [
-              // ...
-            ]
-            // ...
-          }'
-        };
-        let e_thing = WoT.produce(thingDescription);
-        // properties, actions and events are added based on the TD
-        // ...
+      <pre class="example highlight" title="Create a new exposed Thing from a TD URI">
+        // fetch an external TD, e.g., to set up a proxy for that Thing
+        WoT.fetch("http://myservice.org/mySensor/description").then(td => {
+          // WoT.produce() ignores instance-specific metadata (security, form)
```

Comment:
This behavior is not specified yet, so should not be (yet) mentioned in an example.

Reply:
That was an existing example that I only fixed, not add.